crates/sui-indexer/migrations/pg/2024-09-18-003318_epochs_add_bcs_system_state/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE epochs RENAME COLUMN system_state TO system_state_deprecated;
+ALTER TABLE epochs ADD COLUMN system_state bytea;
```

Comment:
not specific to this pr, I realized that a newly added column does not have to be nullable if eventually it is a `NOT NULL`, instead we can just assign it default values while running migrations like
```
ALTER TABLE epochs ADD COLUMN system_state BYTEA NOT NULL DEFAULT E'\\x';
```
this way we can reduce some code change detours.